L1Trigger/TrackFindingTracklet/BuildFile.xml

```diff
@@ -38,6 +39,9 @@
 <use name="TrackingTools/TrajectoryParametrization"/>
 <use name="TrackingTools/TrajectoryState"/>
 <use name="TrackingTools/TransientTrack"/>
+<use name="L1Trigger/TrackerDTC"/>
+<use name="L1Trigger/TrackerTFP"/>
+<use name="SimTracker/TrackTriggerAssociation"/>
```

Comment:
No need to include TrackTriggerAssociation.

Reply:
done